src/components/Wallet/BigCardsGrid/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Container, Grid } from '@mui/material'
+import type { BaseBlock } from '@/components/Home/types'
+import layoutCss from '@/components/common/styles.module.css'
+import getComponentByName from '@/lib/getComponentByName'
+
+const BigCardsGrid = ({ items }: { items: Array<BaseBlock['items'] & { component: string }> }) => (
+  <Container className={layoutCss.containerShort}>
+    <Grid container spacing={{ xs: '30px', xl: '50px' }}>
+      {items?.map((item, index) => {
+        const { component } = item
+
+        const CardComponent = getComponentByName(`${component}`, () => <></>)
```

Comment:
Do we want to fallback to an "empty" component?

```suggestion
        const CardComponent = getComponentByName(component, () => <></>)
```

Reply:
I think it's a suitable fallback. Do you have any other suggestion?

Reply:
I'm not sure - I'll leave it up to you.